Makefile

```diff
@@ -1,24 +1,5 @@
-SKIP_SQUASH?=0
-VERSIONS="5.5 5.6"
+VERSIONS = 5.5 5.6
+OPENSHIFT_NAMESPACES = 5.5
```

Comment:
please comment this.  i had to read through too much scripting to understand what it does.